Sources/Brave/Frontend/Browser/FrequencyQuery.swift

```diff
@@ -83,32 +83,27 @@ class FrequencyQuery {
       if PrivateBrowsingManager.shared.isPrivateBrowsing {
         if let url = tab.url, url.isWebPage(), !(InternalURL(url)?.isAboutHomeURL ?? false) {
           
-          if let selectedTabID = tabManager.selectedTab?.id, let tabID = tab.id, selectedTabID == tabID {
+          if let selectedTabID = tabManager.selectedTab?.id, selectedTabID == tab.id {
             continue
           }
           
-          tabList.append(Site(url: url.absoluteString, title: tab.displayTitle, siteType: .tab, tabID: tab.id))
+          tabList.append(Site(url: url.absoluteString, title: tab.displayTitle, siteType: .tab, tabID: tab.id.uuidString))
```

Comment:
Should we also change Site object use UUID also ?

Reply:
I tried it. It causes me a bunch of problems and is a headache saving CoreData, so I want to do it as a separate PR.